package.json

```diff
@@ -21,6 +21,7 @@
 		"restify": "^4.1.1"
 	},
 	"devDependencies": {
+		"@types/node": "^10.12.14",
```

Comment:
Hum, why ?

No need to add @type... At least not in this PR.

I think you must revert `package` and `package-lock`